hw/arm/virt.c

```diff
@@ -1673,6 +1683,11 @@ static void virt_cpu_post_init(VirtMachineState *vms, int max_cpus,
     bool aarch64, pmu, steal_time;
     CPUState *cpu;
 
+    /* Start logging */
+#ifdef CONFIG_TCG_LOG_INSTR
+    qemu_log_instr_init(CPU(ARM_CPU(first_cpu)));
```

Comment:
This should be called for each CPUState.

Reply:
Also done